Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -160,6 +157,12 @@ private static TDriverOptions SetCommonChromiumOptions<TDriverOptions>(
         // Disabling smooth scrolling to avoid large waiting time when taking full-page screenshots.
         options.AddArgument("disable-smooth-scrolling");
 
+        // Previously this switch caused Chrome processes to remain open after test execution, see
+        // https://github.com/Lombiq/UI-Testing-Toolbox/issues/356, but it doesn't seem to be case anymore.
+        // Additionally, Ubuntu 2024-based GitHub Actions runners seem to require this flag to be set, see
+        // https://github.com/actions/runner-images/issues/8268#issuecomment-2343831000.
+        options.AddArgument("--no-sandbox");
```

Comment:
Note: Local (Windows 10 and Ubuntu 2024) and GHA (GH- and WarpBuild-hosted Ubuntu 2024) runners did not exhibit the previous issue of straggling Chrome processes.